client/state/ui/guided-tours/contexts.js

```diff
@@ -19,15 +20,20 @@ export const isPreviewNotShowing = state =>
 export const isPreviewShowing = state =>
 	isPreviewShowingSelector( state );
 
+const timeSinceUserRegistration = state => {
+	const user = getCurrentUser( state );
+	return user ? ( Date.now() - Date.parse( user.date ) ) : false;
```

Comment:
Sounds like we should attempt to `Date.parse` first, verify we have a number, and then proceed (or bail with `false`), instead of potentially exposing NaNs. Also, is it expectable that `user.date` is in the wrong format for `Date.parse`? If not, we could fix all of this by changing `user ?` into `( user && user.date ) ?`